server/service.go

```diff
@@ -62,6 +62,7 @@ type BoostServiceOpts struct {
 	Relays                []RelayEntry
 	RelayMonitors         []*url.URL
 	GenesisForkVersionHex string
+	GenesisTime           int64
```

Comment:
can be uint64

Reply:
I add a commit for that. However at the end, time.Now().UTC().UnixMilli() is int64 not uint64, so we still need a cast. IMO it would be better to consider all times as int64.